plugins/neo4j-dump-widget/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "datashare-plugin-neo4j-dump-widget",
+  "description": "A Datashare plugin to dump a neo4j graph",
+  "version": "0.2.0",
+  "main": "dist/index.umd.min.js",
+  "dependencies": {
+    "@popperjs/core": "^2.11.8",
+    "axios": "^1.4",
+    "bodybuilder": "^2.5.1",
+    "node-sass": "^8",
```

Comment:
This is [deprecated](https://sass-lang.com/blog/libsass-is-deprecated/), we should use [Dart Sass](https://sass-lang.com/dart-sass/) instead.

Reply:
Also, with `sass-loader`, this is a devDependencies